packages/xstate-inspect/src/browser.ts

```diff
@@ -146,10 +148,10 @@ export function inspect(options?: InspectorOptions): Inspector | undefined {
     stringify(value, options?.serialize);
 
   devTools.onRegister((service) => {
-    const state = service.state || service.initialState;
+    const state = service.getSnapshot() || service.getInitialState();
```

Comment:
This is redundant as `.getSnapshot()` now always calls into `.getInitialState` if needed
```suggestion
    const state = service.getSnapshot();
```